### PR TITLE
Added new contents on my project

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this project, I did the following :
- Added new elements in the HTML file
- Styled all the new elements and used hover on some elements.
- Updated my README file

### CODE REVIEW

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @moise-mulungu 
:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
Your UI looks great! The same as in Figma!

There are that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [x] Did you know that on Figma you are provided with buttons design 😉? I kindly suggest you to add hover effect as provided in Figma:
image

_Check the comments under the review._ 

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -34,5 +34,206 @@ <h2 class="title">I'm a software developer</h2>
...
+                text ever the 1500s, when an<br>
+                unknown printer took a standard dummy<br>
+                text. 
+            </p>
+            <div class="web">
+                <button class="css">css</button>
+                <button class="html">html</button>
+                <button class="boot">boostrap</button>
+                <button class="ruby">Ruby</button>
+            </div>
```
Comment (rotimiazeez) [inline index.html]: - [ ] Please make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") in your works are a list `ul` and each of them a `li`👍

File: index.html [hunk 2]
```diff
@@ -34,5 +34,206 @@ <h2 class="title">I'm a software developer</h2>
...
+                personalized reads: no account or<br>
+                sign-up are required. has been<br>
+                the industry standard
+            </p>
+            <div class="web2">
+                <button class="html">html</button>
+                <button class="boot">boostrap</button>
+                <button class="ruby">Ruby</button>
+            </div>
+            <button class="click2">See Project</button>
```
Comment (rotimiazeez) [inline index.html:90]: - [ ] Please note that the hover effect for the buttons suggested by the previous reviewer has not been implemented🙄, kindly see to it😊

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @moise-mulungu 👍, 

### Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by julie-ify** [comment]
@moise-mulungu Please check your slack messages. I'm chatting with you there but no replies. I want to help you fix the button interactions changes.

**Review by julie-ify — APPROVED** [review]
## Hi @moise-mulungu, 

# Highlights
- Good job implementing the required changes from the previous reviewers ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**